packages/dht/src/connection/MockConnectionManager.ts

```diff
@@ -0,0 +1,25 @@
+import { Message, PeerDescriptor } from '../proto/DhtRpc'
+import { EventEmitter } from 'events'
+import { ITransport, Event } from '../transport/ITransport'
+import { Simulator } from './Simulator'
+
+export class MockConnectionManager extends EventEmitter implements ITransport {
```

Comment:
Could move this and `Simulator` to the simulation package

Reply:
This not a connection manager? Maybe `SimilatorTransport`?

@juslesan suggested that we'd rename simulator directory to kandemlia-simulator to distinguish the "internet simulator" and Kademlia simulator?

Reply:
MockConnectionManager does not exist anymore. Kademlia simulations have been moved to a kademlia-simulation folder